public/contracts/BountyDistribution.sol

```diff
@@ -1,37 +1,183 @@
 pragma solidity ^0.4.21;
 
 contract BountyDistribution {
+    // only employees may alter
+    modifier onlyMembers {
+        require(membership[msg.sender]); 
+        _;
+    }
+
+    // Congress Variables/Structs
+    mapping (address => bool) membership; 
+    mapping (address => MemberProposal) addProposals; 
+    mapping (address => MemberProposal) removeProposals; 
+
+    uint numMembers; 
+    uint debatingPeriodInMinutes = 600; 
+
+    struct MemberProposal { 
+        address recipient; 
+        uint minExecutionDate; 
+        uint numberOfVotes; 
+        int currentResult; 
+        bool open; 
+        bool isValid;
+        
+        mapping (address => bool) voted; 
+    }
+
+    // Q&A Variables/Structs
     struct Question {
         uint256 hash;
         uint256 bounty; // denominated in WEI. 1 ether = 10^18 wei
         address askerAddr;
-        uint256 answerHash;
+        mapping(uint256 => Answer) answers;
+        mapping(uint256 => uint256) indexes; 
+        uint minExecutionDate; 
+
+        uint numAnswers; 
         bool settled;
         bool isValue; // checks for existence of key
     }
 
+    struct Answer {
+        address answerAddr; 
+        uint256 answerHash; 
+        
+        uint256 numUpvotes; 
+        bool isValue;
+        mapping(address => bool) voted; 
+    }
+
     mapping (uint256 => Question) public questions;
-    address contractCreator;
 
-    function BountyDistribution() public { // constructor
-        contractCreator = msg.sender;
+    constructor() public {
+        membership[msg.sender] = true; 
+    }
+
+    // Membership Functions
+    function newProposal(address recipient, bool addAddress) onlyMembers public {
+        if (addAddress) {
+            require(!addProposals[recipient].isValid || !addProposals[recipient].open);
+        } else {
+            require(!removeProposals[recipient].isValid || !removeProposals[recipient].open);
```

Comment:
require member to be a part of the organization already in order for them to be removed